cmd/pilot-agent/main.go

```diff
@@ -220,6 +247,9 @@ func init() {
 		"IP Address and Port of a statsd UDP listener (e.g. 10.75.241.127:9125)")
 	proxyCmd.PersistentFlags().IntVar(&proxyAdminPort, "proxyAdminPort", int(values.ProxyAdminPort),
 		"Port on which Envoy should listen for administrative commands")
+	proxyCmd.PersistentFlags().IntVar(&infraAuthPolicy, "infraAuthPolicy", int(values.InfraAuthPolicy),
```

Comment:
How about a boolean flag 'allowInsecure' ( to make it clear what it does - it's not really a policy ).

Reply:
Maybe allowInsecurePolicy